src/lib/i18n.js

```diff
@@ -0,0 +1,38 @@
+import i18n from 'i18next'
+import { initReactI18next } from 'react-i18next'
+
+const resources = {

```

Comment:
Great use case for types. You can define a type that expects x keys for every language. That way, you won't miss strings in other languages.

Reply:
I figured it would be better to fallback for the strings not populated. However I can try this if you prefer, please clarify

Reply:
I think there's very few cases where we'd intentionally not fill in the strings for another language, unless I'm mistaken.

Im particularly thinking about cases where we may just forget to add the translations being more common.

Does this sound reasonable?